packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -143,16 +254,13 @@ const handler = async (event) => {
   return Promise.all(sqsMessages.map(async (message) => {
     const executionEvent = parseSQSMessageBody(message);
     const cumulusMessage = await getCumulusMessageFromExecutionEvent(executionEvent);
-    const results = await Promise.allSettled([
-      saveExecution(cumulusMessage, knex),
-      saveGranulesToDb(cumulusMessage),
-      savePdrToDb(cumulusMessage),
-    ]);
-    if (results.some((result) => result.status === 'rejected')) {
+
+    try {
+      await saveRecords(cumulusMessage, knex);
+    } catch (error) {
```

Comment:
I believe that my implementation preserves the existing behavior where a failure in writing any of the records (to just Dynamo or Dynamo/RDS) results in the message being written to the DLQ. The unit test for this behavior was previously named to suggest that the DLQ was only written to if granule and PDR writes to the database failed, but the actual logic would write when **any failure to write** was encountered:

https://github.com/nasa/cumulus/blob/master/packages/api/tests/lambdas/test-sf-event-sqs-to-db-records.js#L293
https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/sf-event-sqs-to-db-records.js#L57

I kept the unit test for this behavior as previously implemented but updated the name accordingly

Reply:
Good catch - I believe that was indeed the original intent.